todo_app/app.py

```diff
@@ -8,11 +8,23 @@
 
 from todo_app.flask_config import Config
 from todo_app.data.viewmodel import ViewModel
+import logging
 
+from loggly.handlers import HTTPSHandler
+from logging import Formatter
 
 
 app = Flask(__name__)
 app.config.from_object(Config())
+if app.config['LOGGLY_TOKEN'] is not None:
+    handler = HTTPSHandler(f'https://logs-01.loggly.com/inputs/{app.config["LOGGLY_TOKEN"]}/tag/todo-app')
+    handler.setFormatter(Formatter("[%(asctime)s] %(levelname)s in %(module)s: %(message)s")
+    )
+    app.logger.addHandler(handler)
+app.logger.info('This a debug message')
+logging.warning('This an info message')
+logging.error('This an error message')
+logging.critical('This a criticl message')
```

Comment:
This works, but its not particularly helpful, and is only sending the first of these to Loggly which feels incorrect!

If we want to log the other messages, we similarly need to use `app.logger.<log_level>` like `app.logger.critical('This is a critical message')`

It would be good to add some more useful logging, such as:
* A log when a new item is added or...
* A log when an item is deleted, like:
```
@app.route('/delete_card', methods=["POST"])
def delete_item():
    item = request.form["delete_card"]
    delete_card(item)
    app.logger.info(f"Item {item} was deleted")
    return redirect(url_for("home"))

```

Reply:
Thank you Jack, I have completed this now and have been receiving useful logs from Loggly
![Screenshot 2023-01-26 at 09 14 58](https://user-images.githubusercontent.com/78435996/214799483-a9ee6cec-04e2-4f8a-b56f-ffbc6795e5ce.png)